java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -1119,6 +1119,7 @@ public void non_UTF8_GlideString_map_with_geospatial(BaseClient client) {
     @ParameterizedTest(autoCloseArguments = false)
     @MethodSource("getClients")
     public void non_UTF8_GlideString_map_of_arrays(BaseClient client) {
+        assumeTrue(SERVER_VERSION.isGreaterThanOrEqualTo("7.0.0"));
```

Comment:
Can you split please test fixes and CI fixes into 2 PRs?

Reply:
Java code is a cherry-pick, it will disappear when you will backport fixes 
I just want to get to a point i see that its green and no extra fixes needed